.rubocop.yml

```diff
@@ -50,6 +50,10 @@ Style/CommentedKeyword:
 Style/EmptyCaseCondition:
   Enabled: false
 
+Style/RedundantInitialize:
+  Exclude:
+    - test/mock_helpers/arel.rb
+
```

Comment:
This solves the bug in rubocop that complains for the "redundant initializer" and also complains for the `Unnecessary disabling of Style/RedundantInitialize` if you disable it with inlining `# rubocop:disable Style/RedundantInitialize` 🤷

In the `v6` branch I added a [dummy line](https://github.com/ddnexus/pagy/blob/0cc6b06ecc4951568ffb4a5b50bd342d21b1a5a3/test/mock_helpers/arel.rb#L19) in order to avoid the global exclude of a single file, wondering which one is "better"

Reply:
@ddnexus Just let me know how you'd prefer to handle it and I can make any change necessary.  I don't have a strong preference here.

Reply:
I'll go with yours. 

Please, could you revert my dummy line, and add some comment like `# cannot inline the disabling in the file because rubocop complains anyway` comment on top of the exclude in `.rubocop.yml`. Thank you. 

_(I tried to do it myself rebasing your branch on `v6` but when I tried to push it to your branch, there was a long list of commits added below yours, and I didn't wanted to mess up your branch. Github `Squash and merge` will sort it out.)_

Reply:
@ddnexus I've made these changes.  Please let me know if anything else is required.  Thanks.